src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -921,23 +921,20 @@ protected function getColumnOverrides() {
       'weight',
       'class',
     ];
-    $column_overrides_sorted = [];
-    $pair = 2;
 
-    foreach ($this->config('data', 'column_overrides') as $field_name => $column_override) {
-      $field_overrides = explode(PHP_EOL, $column_override);
-
-      foreach ($field_overrides as $field_override) {
-        $field_override_array = explode('|', trim($field_override), $pair);
+    $column_overrides = array_fill_keys($this->fieldLabelsOriginal(), []);
 
-        if (count($field_override_array) === $pair && in_array($field_override_array[0], $allowed_overrides)) {
-          $column_overrides_sorted[$field_name][$field_override_array[0]] = $field_override_array[1];
-        }
+    foreach ($this->config('data', 'column_overrides') as $field_name => $column_override) {
+      if (empty($column_override)) {
+        continue;
       }
+
+      $real_field_name = substr($field_name, 1);
+      $column_overrides[$real_field_name] = $this->dvfHelpers->configStringToArray($column_override);
     }
 
-    $column_overrides_sorted = array_merge(array_fill_keys($this->fieldLabelsOriginal(), []), $column_overrides_sorted);
-    return $this->setArrayOrder($column_overrides_sorted);
+    $sorted = $this->setArrayOrder($column_overrides);
+    return $sorted;
```

Comment:
probably don't need the `$sorted` variable here

Reply:
tnx @thery-dh you found my debugging line ;) 
Updated now